code/modules/vehicles/mecha/equipment/tools/work_tools.dm

```diff
@@ -95,7 +96,7 @@
 				chassis.visible_message(span_notice("[chassis] pushes [target] out of the way."), \
 				span_notice("[chassis] pushes you aside."))
 			return ..()
-		else if((source.istate & ISTATE_SECONDARY) && iscarbon(M))//meme clamp here
+		else if(LAZYACCESS(modifiers, RIGHT_CLICK) && iscarbon(M))//meme clamp here
```

Comment:
revert this back to ISTATE_SECONDARY, as modifiers right click isn't guaranteed to be secondary actions anymore

Reply:
oh whoops! I don't think this was intended at all, something happened with GH that resulted in a lot of mess, oops